tests/QssTestDefs.js

```diff
@@ -1362,6 +1400,12 @@ var quickFoldersTestSequence = [
         ],
         resolve: "jqUnit.assertTrue",
         resolveArgs: ["The quick folder button is displayed: ", "{arguments}.0"]
+    }, {   // XXX we need some minor timeout for the QSS to get
+        // in normal state. In case this is not present,
+        // the next item doesn't take effect
+        task: "gpii.test.linger",
```

Comment:
Please figure out what event you need to wait for and add a listener for it instead of this long timeout

Reply:
Bump comment

Reply:
@krisYanachkov can you take a look on this, because its the last part of this PR.